### DISCUSSION TITLE
I don't want to change my button color to blue on pressing button.

### DISCUSSION THREAD

**Discussion by leamandeep** [saadeghi/daisyui#1348] [Q&A]
I don't want to change my button color to blue on pressing button. Any help?

![image](https://user-images.githubusercontent.com/68345524/201454892-29107311-bf62-457f-91ed-c51b9784d260.png)

**Comment by anhsirk0** [comment]
You can use `active:bg-transparent` on the `a` tag. 
If it does'nt help, provide a code snippet for the Drawer content.

**Comment by leamandeep** [comment]
Not working . Purple color not gone on pressing button

```html
 <ul id="shapes"
            tabindex="0"
            class="dropdown-content menu menu-horizontal active:bg-green-400 p-2 text-slate-900 shadow-xl bg-base-100 
rounded-box w-52 z-20"
          >
            <li
              class="tooltip tooltip-accent tooltip-bottom font-semibold capitialize z-20 active:bg-blue-700"
              data-tip="Line"
            >
              <label class="relative flex justify-center items-center">
                <input
                  type="radio"
                  class="opacity-0 absolute w-10 h-10 z-20"
                  name="shapes"
                  value="line"
                />
                <i class="fa-solid fa-arrows-left-right"></i>
              </label>
            </li>
</ul>
```

![image](https://user-images.githubusercontent.com/68345524/201503071-b153eaf1-9e0d-4d3a-baea-2928a1a9731c.png)

**Comment by anhsirk0** [comment]
Try `active:bg-transparent` on the `label` tag

**Reply by leamandeep** [reply]
Not working even in menu 😢😢

**Reply by anhsirk0** [reply]
I could not reproduce this error. Can provide a repo/codepen.
